### PR TITLE
Lab2

### PR DESCRIPTION
**Pull request by MikhailMarch**

### CODE REVIEW

**Comment by al1ych** [comment]
Typo:
"In my Dockerfile I used COPY over ADD as COPY is more predictable and less error prone."
Should be
"error prone"

**Comment by al1ych** [comment]
docker image works fine

**Comment by emileyray** [comment]
Looks nice

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ 1)  First, install [docker](https://docs.docker.com/engine/install/).
+
+Building image locally:
+ 1) change the directory to the app_python folder
+ 2) then use `docker build -t time_application`
+ 3) then use `docker run -p 5000:5000 -d time_application`
+ 4) Navigate to the following [link](127.0.0.1:5000) to check
+
+Build from docker hub image
+ 1) Run command `docker run -p 5000:5000 -d mikhailmarch/time_application'
```
Comment (SMore-Napi) [inline README.md]: The last symbol should be \` instead of '

**Comment by Dablup** [comment]
Tested with docker image, works well

**Pull request closed without merging**